__tests__/_helper.ts

```diff
@@ -104,6 +108,10 @@ export function returnJson(json: Record<string, unknown>): RestResolver {
   return (_req, res, ctx) => res.once(ctx.json(json))
 }
 
-export function returnApiUuid(uuid: unknown): RestResolver {
+export function apiToReturnError(body: string, { status = 404 }): RestResolver {
+  return (_req, res, ctx) => res.once(ctx.status(status))
+}
```

Comment:
```suggestion
export function apiToReturnError({ status = 404 }): RestResolver {
    global.server.use(rest.post(global.API_ENDPOINT, (_req, res, ctx) => res.once(ctx.status(status))))
}
```

or

```typescript
export function apiToReturnError(url: string, { status = 404 }): RestResolver {
    global.server.use(rest.post(url, (_req, res, ctx) => res.once(ctx.status(status))))
}
```

For the first you need to call it like when `global.API_ENDPOINT` is not set somewhere else:

```typescript
global.API_ENDPOINT = "http://..."
apiToReturnError()
```

For the second you have:

```typescript
apiToReturnError("http//...")
```